--- origsrc/Parse-Yapp-1.05/Makefile.PL	1999-07-11 11:38:07.000000000 -0500
+++ src/Parse-Yapp-1.05/Makefile.PL	2008-04-08 23:45:32.171875000 -0500
@@ -4,7 +4,6 @@
 WriteMakefile(
     'NAME'	=> 'Parse::Yapp',
     'VERSION_FROM' => 'lib/Parse/Yapp/Driver.pm', # finds $VERSION
-    'MAN3PODS' => { 'lib/Parse/Yapp.pm' => '$(INST_MAN3DIR)/Parse::Yapp.3' },
     'PM' => {
               'lib/Parse/Yapp.pm' => '$(INST_LIBDIR)/Yapp.pm',
               'lib/Parse/Yapp/Options.pm' => '$(INST_LIBDIR)/Yapp/Options.pm',
